google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1/error_stats_service_client.rb

```diff
@@ -310,9 +308,9 @@ def initialize \
 
           def list_group_stats \
               project_name,
-              time_range,
               group_id: nil,
               service_filter: nil,
+              time_range: nil,
```

Comment:
@dazuma This PR changes a positional param to keyword arg in the lower-level API.